### PR TITLE
Note requirement of go 1.16 to build

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
    with no HyperShift installed yet.
 
 2. Build HyperShift.
-
+        # requires go v1.16+
```
Comment (csrwng) [inline HACKING.md:11]: @bparees if you look at a preview of this page, this doesn't look right. Maybe the requires comment should go in the line above where it is.

Comment (bparees) [reply HACKING.md:12]: fixed

**Pull request merged**